.github/workflows/ci_testing.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix: ${{ fromJSON(needs.check-diff.outputs.matrix) }}
-    timeout-minutes: 35
+    timeout-minutes: 90
```

Comment:
why did it increase so much? Or is this just some higher value? if so, can we try to find out the actual maximum so we know when we have a regression?

Reply:
we never run full TM tests here only integrations to PL (5min top) and from TM we know that the full test suit on Mac takes about 75min :(